src/main/java/com/skynest/tests/files/RegistrationPayload.java

```diff
@@ -0,0 +1,15 @@
+package com.skynest.tests.files;
+
+import lombok.*;
+@Setter @Getter
+@AllArgsConstructor
+@NoArgsConstructor
+
+public class RegistrationPayload {
```

Comment:
1. again, technically speaking, each http request will have some sort of payload - the RegistrationResponse class also is a model of the payload delivered to us from the api. Therefore, it might be preferable to call this class "RegistrationRequest", especially given it's appropriately evocative of the request-response model. 

2. also, consider having stricter modifiers on the class/fields. Since this class will capture a response from an api, which we practically never want to tamper with;  we might want to make its fields final as well and remove the setters

Reply:
Makes sense to stick to one consistent naming pattern for the payloads from the http requests.